### DISCUSSION TITLE
Re-IP Network Issue

### DISCUSSION THREAD

**Discussion by quella** [FreshRSS/FreshRSS#3570] [General]
For reasons, I was required to re-IP my local network from the 192.168.1.xxx to the 192.168.99.xxx.  Everything appears to be working in the application apart from the API which is still listing the older IP Address (192.168.1.54) instead of the newly assigned 192.168.99.54; see the image below.  

When I look at the "Profile" section via the WebGUI, I see that the older IP is still in the API query info, etc.  I did a reboot and that did not seem to correct the issue.  Is there an easy way to update or modify this via a config file or other means?  I tried to click on some of the icons next to them, but I do not see a way of correcting it.

![Screen Shot 2021-03-30 at 11 10 31 AM](https://user-images.githubusercontent.com/823352/113016201-4371b700-914c-11eb-8aec-90b4205cd60f.png)

Here is a copy showing the WebUI and the new IP working as expected:
![Screen Shot 2021-03-30 at 11 49 39 AM](https://user-images.githubusercontent.com/823352/113018272-5c7b6780-914e-11eb-9a9e-459ba3ed9427.png)

**Comment by Alkarex** [comment]
Hello,
Please adjust `base_url` in `./data/config.php` and use a DNS if possible

**Reply by quella** [reply]
Sir, it worked like a charm.  Thank you for the pointer on the config file.  Made the change, restarted and all is back to normal.  Working on internal DNS, that is my next project.  :)
